compose/install.md

```diff
@@ -34,7 +34,7 @@ To install Compose, do the following:
     The following is an example command illustrating the format:
 
     ```bash
-    $ curl -L "https://github.com/docker/compose/releases/download/1.11.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
Leaving the `--fail` flag at the end makes it a bit difficult to read the command. Can you move it to the beginning, to be like `curl -L --fail ...`